### ISSUE TITLE
New rules for orthography profiles to increase explicitness and transparency

### ISSUE THREAD

**Issue by chrzyki** [lexibank/lexibank#105]
_From @LinguList on January 29, 2018 10:9_

Our usage of orthoprofiles is lacking one step of explicitness. So far, we have a column "grapheme" and an IPA column. But often we may have a grapheme that will later be "unmerged" by adding a space between the symbols in IPA. This secondary segmentation is a huge relieve for orthoprofile creation and drastically facilitates our work. However, in order to keep track of the original graphemes, we would ideally add another column where we only segment the graphemes properly. We could call it "segments" for the time being, or whatever. But we would profit from this, as it would give us direct pairs of source-target sounds, which we could later feed in systems like new orthoprofiles, or CLTS.

_Copied from original issue: glottobank/lexibank-data#359_

**Comment by tresoldi** [comment]
Is this issue still relevant? In particular, it does not seem related to orthographic profiles properly (i.e., we would not change any of the existent profiles), but to pylexibank/cldfbench.

As for keeping the original grapheme, only segmented, @LinguList has a good point, and it could help winning "traditional" linguists (such as those insisting on their graphemes but still wanting segments). Still, I wonder if this is not just a matter of mapping `Grapheme` to `Grapheme` itself, equivalent to manually overriding the `IPA` values with those in `Grapheme`, getting a dictionary where keys and values match, and generating a new string of space separated segments.

**Comment by LinguList** [comment]
We discussed the possibility of adding an internal grapheme segmentation inside Grapheme, e.g., by adding a gap character, so `^khw` maps to the ipa equivalent of `k_h w` and the grapheme equivalent of `kh w`, e. g., by writing `^kh-w` for Graphemes,  but as we use start and stop chars as literal symbols, we would not like to have them inside this grapheme, and I wonder how tedious and useful this practice is ingeneral. In any case, it would allow us to apply a sanity check for all profiles, by forcing the length of graphemes (dash-segmented) to be identical with the length of IPA (space-segmented).

**Comment by tresoldi** [comment]
I see, it seemed to me that the goal was to have such check in the segmented data (e.g., in `forms.csv`), not in the profile itself.

But I also wonder how tedious and useful it would be, besides adding an overall complexity. Should we than close this issue? Other team members?

**Comment by LinguList** [comment]
How do you want to check that in a segmented data file? 

The essential message here is: we abuse orthoprofiles for our purpose of getting plain IPA, but the original idea was that the Graphemes column lists graphemes, not sequences that are split secondarily. 

If we go for a larger orthograpy, I assume it is essential to offer more than jus the conversion to IPA, as it is not always one-to-one, but if we add a column Segments or similar, containing segmented graphemes, this would address the problem. It is an additional quality of a given profile. I would not close it, but opt for discussing it with future versions and an extension of orthograpy and the like.

And my preferred solution is in fact: having an additional column, no inline-segmentor in the original Graphemes, as Graphemes is listing raw data, probably badly typed off, while GraphemeSegments would list the data as it was intended in a given source (without typos resulting from digitization etc.).

**Comment by tresoldi** [comment]
I would have a column "Grapheme_Segments" with the graphemes separated by spaces, as the result of running a mapping Grapheme->Grapheme instead of Grapheme->IPA. But to be clear, I am not suggesting this, it what I understood was being proposed.

Let's wait for the others to comment; we can always agree to move the issue from Lexibank 1.0 to a future version.

**Comment by LinguList** [comment]
Yes, that's what I proposed, and what I think would be the most useful also for any potential publication of orthograpy.
